substrate/runtime-std/with_std.rs

```diff
@@ -25,3 +25,4 @@ pub use std::ptr;
 pub use std::rc;
 pub use std::slice;
 pub use std::vec;
+pub use std::collections::hash_map;
```

Comment:
could have a `pub mod collections` which exports the things we want. mirroring the structure of `std` and `alloc` as closely as possible (`rstd::collections::HashMap` as opposed to `rstd::hash_map::HashMap`) will be easier to use

Reply:
i thought you might say that...